templates/stripe-payment.yaml

```diff
@@ -68,22 +70,22 @@ code:
     });
 
     function renderView(ctx) {
+      //change /stripe-payment below to your task name
       return `
-      <form action="/stripe/payment?currency=USD&amount=2000&description=Test%20item" method="POST">
+      <form action="/stripe-payment/payment?currency=USD&amount=2000&description=Test%20item" method="POST">
```

Comment:
Don't you normally include this info in the body of the form post (i.e. via the data attributes below)?